__init__.py

```diff
@@ -96,6 +96,70 @@ def contentdir(self):
         return 'github-ci'
 
 
+class SpicyAnalyzer(zeekpkg.template.Feature):
+    """Feature for a Spicy-based analyzer."""
+
+    def contentdir(self):
+        return 'spicy-analyzer'
+
+    def needed_user_vars(self):
+        """Specify required user variables."""
+        return ['name', 'namespace']
+
+    def validate(self, tmpl: zeekpkg.template.Template):
+        """Validate feature prerequisites."""
+        for parameter in ['name', 'ns']:
+            value = tmpl.lookup_param(parameter)
+            if not value or len(value) == 0:
+                raise zeekpkg.template.InputError(
+                        'package requires a {}'.format(parameter))
+
+    def instantiate(self, tmpl):
+        super().instantiate(tmpl)
+
+        def pkg_file(*name):
+            p = os.path.join(self._packagedir, *name)
+            assert os.path.exists(p)
+            return p
+
+        # Manually merge Spicy analyzer-specific changes to `zkg.meta`.
+        with open(pkg_file('zkg.meta'), 'ab') as f:
+            # Add to the `depends` entry.
+            f.write(b'\tspicy-plugin\n')
+
+            # Add a write command.
+            f.write(b'build_command = unset -v CXX CXXFLAGS LD LDFLAGS && mkdir -p build && cd build && cmake .. && cmake --build .\n')
```

Comment:
Benjamin I've always been curious about this — why the unset? It seems unusual, because normally you want to permit such adjustments to get passed down when necessary.

Reply:
If these variables are expected to customize the behavior of e.g., plugins installed through `zkg` you are right that this `unset` should not be there. I removed it.